Yoast/yoast-components
# Decode separator replacement variable

## Summary

This PR can be summarized in the following changelog entry:

* Decodes the separator replacement variable before it is passed along to the renderer

## Relevant technical choices:

* Finds the replacement variable `sep` and decodes the HTML code to a displayable symbol

## Test instructions

This PR can be tested by following these steps:

* Change the `sep` replacement variable to different symbols and check if they are displayed properly in the title and the description (no escaped HTML codes). The `sep` replacement variable can be changed in your local wp-admin sidebar by going to SEO > Search Appearance > General.

Fixes https://github.com/Yoast/wordpress-seo/issues/9787
